fe/fe-core/src/main/java/com/starrocks/load/routineload/RoutineLoadTaskScheduler.java

```diff
@@ -84,7 +84,7 @@ public class RoutineLoadTaskScheduler extends FrontendDaemon {
     private final RoutineLoadMgr routineLoadManager;
     private final LinkedBlockingQueue<RoutineLoadTaskInfo> needScheduleTasksQueue = Queues.newLinkedBlockingQueue();
     private final ScheduledExecutorService scheduledExecutorService = Executors.newSingleThreadScheduledExecutor();
-    private final ExecutorService threadPool = Executors.newFixedThreadPool(THREAD_POOL_SIZE);
+    private final ExecutorService threadPool = Executors.newCachedThreadPool();
```

Comment:
remove `THREAD_POOL_SIZE` variable

Reply:
The max thread pool size is Integer.MAX_VALUE, is it OK?

Reply:
The thread pool size is not bigger than be number * max_routine_load_task_num_per_be. So, it's limited indeed.